op-node/rollup/driver/state.go

```diff
@@ -200,11 +199,12 @@ func (s *Driver) eventLoop() {
 		sequencerTimer.Reset(delay)
 	}
 
-	// Create a ticker to check if there is a gap in the engine queue every 15 seconds
-	// If there is, we send requests to the backup RPC to retrieve the missing payloads
-	// and add them to the unsafe queue.
-	altSyncTicker := time.NewTicker(15 * time.Second)
+	// Create a ticker to check if there is a gap in the engine queue, whenever
+	// If there is, we send requests to sync source to retrieve the missing payloads.
```

Comment:
```suggestion
	// Create a ticker to check if there is a gap in the engine queue. Whenever
	// there is, we send requests to sync source to retrieve the missing payloads.
```

Reply:
This didn't get changed either.  Either something went wrong with rebasing or my GitHub is broken...